docs/API.md

```diff
@@ -198,6 +202,15 @@ credentials than were specified to [`RFB()`](#rfb-1). The `detail`
 property is an `Object` containing the property `types` which is an
 `Array` of `DOMString` listing the credentials that are required.
 
+#### securityfailure
+
+The `securityfailure` event is fired when the handshaking process with
+the server fails during the security negotiation step. The `detail`
+property is an `Object` optionally containing the property `reason`
+which is a `DOMString` specifiying the reason for the failure. The
+server can choose to not send a reason and in these cases the property
+`reason` will be omitted.
```

Comment:
Can you try to make it clearer that this is a server provided message, rather than something internal? Perhaps also mention that the language is not fully known (although most likely english)?

Reply:
Yes, will do